pkg/csi/cinder/openstack/openstack_volumes.go

```diff
@@ -222,6 +223,35 @@ func (os *OpenStack) WaitDiskAttached(instanceID string, volumeID string) error
 	return err
 }
 
+//WaitVolumeStatusAvailable waits for volume to be 'available'
```

Comment:
This code looks good, but it may be worth changing the comment to indicate that this only works for volumes moving directly from the "creating" state to the "available" state.

(This is always true for how WaitVolumeStatusAvailable is used here, but it may not be true for someone using it in a different context later.)

Reply:
+1, Ill update